llpc/context/llpcCompiler.cpp

```diff
@@ -2549,8 +2544,6 @@ Result Compiler::buildRayTracingPipelineInternal(RayTracingContext &rtContext,
     helperThreadProvider->SetTasks(&helperThreadBuildRayTracingPipelineElf, newModules.size(),
                                    static_cast<void *>(&helperThreadPayload));
     {
```

Comment:
This scope looks like it was only intended for the guard. Also for some of the other guards.

Reply:
Good point.